content/blog/2022-12-15-introduction-to-the-ethereum-classic-course/index.md

```diff
@@ -81,7 +81,7 @@ Below are the classes with the topics that we will touch in each section.
 37. DAOs
 38. Contracts Between Individuals and Businesses on the Blockchain
 
-As seen in this introduction the Ethereum Classic course promises to be a thorough educational experience about ETC and the industry in general and is divided in several sections with small units so they are easier to process and comprehend.
+As seen in this introduction, the Ethereum Classic course promises to be a thorough educational experience about ETC and the industry in general, and it is divided in several sections with small units so they are easier to process and comprehend.
```

Comment:
```suggestion
As seen in this introduction, the Ethereum Classic course promises to be a thorough educational experience about ETC and the industry in general. It is divided into several sections with small units so they are easier to process and comprehend.
```
The current revision is a comma-spliced run-on sentence; a "no-no" in written English.